alert-engine/src/test/scala/io/phdata/pulse/alertengine/AlertEngineImplTest.scala

```diff
@@ -57,16 +58,30 @@ class AlertEngineImplTest extends FunSuite with BaseSolrCloudTest with MockitoSu
 
     val document = DocumentConversion.toSolrDocument(
       LogEvent(Some("id"),
-               "ERROR",
+               "ALERT",
                "1970-01-01T00:00:00Z",
                "ERROR",
                "message",
                "thread oxb",
                Some("Exception in thread main"),
                None))
 
+    val document2 = DocumentConversion.toSolrDocument(
```

Comment:
Can we get rid of one of these documents? Why document and document2?

Reply:
I think we'd better keep two documents here, because document is the one which doesn't match the query "category:ERROR", and the rest of 12 document2s match the query. So we can verify that it's getting all the correct results if it returns 12 not 13.
I can change the names of the two documents for better translation.